PhysicsTools/NanoAOD/python/nanoDQM_cfi.py

```diff
@@ -593,6 +593,7 @@
                 Count1D('_size', 9, -0.5, 8.5, 'slimmedPhotons after basic selection (pt > 5 )'),
                 Plot1D('charge', 'charge', 1, -0.5, 0.5, 'electric charge'),
                 Plot1D('cleanmask', 'cleanmask', 1, 0.5, 1.5, 'simple cleaning mask with priority to leptons'),
+                Plot1D('cutBased_RunIIIWinter22','cutBased_RunIIIWinter22', 4, -0.5, 3.5,'cut-based ID bitmap, RunIIIWinter22, (0:fail, 1:loose, 2:medium, 3:tight)'),
```

Comment:
This should be changed to `cutBased_Fall17V2`, following the change in the table

Reply:
Naming convention have been changed. The way it is done for the mentioned PR